<ui version="4.0" >
 <class>ossimPlanetQtWmsDialog</class>
 <widget class="QDialog" name="ossimPlanetQtWmsDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WMS Server definitions</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Server Connections</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QComboBox" name="theServerConnections" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QPushButton" name="theConnectServerButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>C&amp;onnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theNewServerButton" >
          <property name="text" >
           <string>&amp;New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theEditServerButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theDeleteServerButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theAddDefaultServersButton" >
          <property name="statusTip" >
           <string>Adds a few example WMS servers</string>
          </property>
          <property name="whatsThis" >
           <string comment="Adds several example WMS servers to the list" />
          </property>
          <property name="text" >
           <string>Add default servers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theWmsSynchButton" >
          <property name="statusTip" >
           <string>Adds a few example WMS servers</string>
          </property>
          <property name="whatsThis" >
           <string comment="Adds several example WMS servers to the list" />
          </property>
          <property name="text" >
           <string>Sync</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="theWmsSynchAllButton" >
          <property name="statusTip" >
           <string>Adds a few example WMS servers</string>
          </property>
          <property name="whatsThis" >
           <string comment="Adds several example WMS servers to the list" />
          </property>
          <property name="text" >
           <string>Sync all</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>141</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="theCacheDirLabel_2" >
       <property name="text" >
        <string>Cache Dir:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="theCacheDir" />
     </item>
     <item>
      <widget class="QPushButton" name="theCacheDirButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="theGroupImageEncoding" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>64</height>
      </size>
     </property>
     <property name="title" >
      <string>Image encoding</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Layers</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTreeWidget" name="theLayersList" >
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="columnCount" >
         <number>4</number>
        </property>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>3</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="theAddButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="theCloseButton" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>theServerConnections</tabstop>
  <tabstop>theConnectServerButton</tabstop>
  <tabstop>theNewServerButton</tabstop>
  <tabstop>theEditServerButton</tabstop>
  <tabstop>theDeleteServerButton</tabstop>
  <tabstop>theAddDefaultServersButton</tabstop>
  <tabstop>theCacheDir</tabstop>
  <tabstop>theCacheDirButton</tabstop>
  <tabstop>theLayersList</tabstop>
  <tabstop>theAddButton</tabstop>
  <tabstop>theCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
